### DISCUSSION TITLE
İbax  Puan Kasma Yolları...

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#882] [Bounty Hunter]
Herkese Merhaba Arkadaşlar Sizin için nasıl ibax puan kasılır derledim :) :) 

1-) IBAX puanları almak için IBAX etkinliklerine ve GitHub tartışmalarına katılın

--- Twitter Üzerinden çekilişler düzenleniyor şansınızı deneyim derim :)  GitHub da Aktif olun ve sohpet edin

2-) GitHub  Yapıcı konular önerin

--- Onaylanan her yapıcı konu 200 puan ile ödüllendirilecek ve ödül her ayın 5'inden sonraki bir hafta içinde (IBAX günü) verilecektir. Teknik araştırma değeri, iş başvurusu değeri, pazar geliştirme beklentileri, akademik keşif değeri ve diğer konular varsa, ibax Vakfı bunu bir ödül görevine dönüştürecek, ayrıca IBAX Puanları da IBXC Coins ile ödüllendirilecektir.

3-) IBAX güvenlik açıklarını bulun

--- IBAX blok zincirindeki hataları bulun, yeniden üretim sürecini gönderin, IBAX ekibi tarafından doğrulandıktan sonra 1~10.000 IBXC ödülü ve IBAX puanı ödülü alabilirsiniz. IBAX ekibi tarafından kabul edilen veya projeyle birleştirilen hatalara kod çözümleri göndermek, IBAX puanlarının yanı sıra 10~50.000 IBXC ile ödüllendirilecektir. IBAX blok zincirinin performansını ve verimliliğini artırabilecek önemli optimizasyon kodu bulursanız, 10~1.000.000 IBXC ve IBAX puanı ile ödüllendirileceksiniz.

4-) Ödül Sorunlarını Çözün

--- IBAX ödül alanına gidin, ödül sorunlarını çözün ve IBXC jetonları ve IBAX puan ödülleri kazanın.
                        Link=https://github.com/IBAX-io/go-ibax

Şu anda Puan kasmak bunlarla mümkündür Yeni şeyler eklendiği zaman size bilgi vereceğim
Hoşça kalın...

**Comment by ghost** [comment]
Eline Sağlık kardeşim çok açıklayıcı olmuş

**Reply by Skips1** [reply]
Sağol kardeşim

**Reply by Omertastan2036** [reply]
Knk bu ödül sorunlarını çözün ne oluyor
